exporter/chronicleexporter/config.go

```diff
@@ -55,6 +55,9 @@ type Config struct {
 	// LogType is the type of log that will be sent to Chronicle.
 	LogType string `mapstructure:"log_type"`
 
+	// OverrideLogType is the field name that will be used to override the log type.
+	OverrideLogType bool `mapstructure:"override_log_type"`
```

Comment:
Comment indicates this is a field name, but this is a bool

Reply:
Could we also add this field to the README's table?

Reply:
updated comment and added to readme, was originally a field but we decided to just have a set val